include/rocksdb/write_batch.h

```diff
@@ -100,7 +100,8 @@ class WriteBatch : public WriteBatchBase {
     return Put(nullptr, key, value);
   }
 
-  // UNDER CONSTRUCTION -- DO NOT USE
+  // Store the mapping "key->{column1:value1, column2:value2, ...}" in the
+  // column family specified by "column_family".
```

Comment:
in the write batch?

Reply:
These comments are actually "inspired by" (i.e. shamelessly copied from) the comments for `Put` ("Store the mapping "key->value" in the database."). I guess the idea there was to describe the end goal of the operation; I changed database to column family because I felt it was more precise. Let me known if you prefer some other wording.

Reply:
Thanks for the clarification, I am fine with it.